index.js

```diff
@@ -1,18 +1,50 @@
 /***** Deliverable 1 *****/
-const header = document.querySelector("#header")
+const header = document.querySelector("h1#header")
 console.log("Here's your header:", header)
-
+//Had to change the index.js script to defer afterwards
 
 /***** Deliverable 2 *****/
 
-
+header.style.color = "blue"
 
 /***** Deliverable 3 *****/
-// console.log('This is what the traveler object looks like: ', traveler)
+console.log('This is what the traveler object looks like: ', traveler)
+//Available because linked in index.html file with: <script src="data.js"></script>
+const travName = document.querySelector('h2')
+const travNick = document.querySelector('em')
+const travPic = document.querySelector('img')
+travName.textContent = traveler.name
+travNick.textContent = traveler.nickname
+travPic.src = traveler.photo
```

Comment:
awesome, would like the variables to be more specific,

`travNameHeader`
`travNicknameEm`
`travPicImg`

Also don't forget to add the alt attribute for all your images!!

`travPic.alt = traveler.name`